### PR TITLE
Prabha k

### PR DESCRIPTION
**Pull request by prabha0710**

### CODE REVIEW

File: PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html
Comment (msindia-learning) [inline PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html:1]: h2, h6?

File: PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Paragraphs.html
Comment (msindia-learning) [inline PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Paragraphs.html:1]: Html template was missing

File: PrabhaK/01 - HTML/01 - Basics/02 - Quotations & Comments/Quotations.html
Comment (msindia-learning) [inline PrabhaK/01 - HTML/01 - Basics/02 - Quotations & Comments/Quotations.html:1]: blockquote

File: PrabhaK/01 - HTML/01 - Basics/03 - Links & Images/Images.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!--
+    HTML Images
+-->
+<img src="url" alt="alternate text" />
+
+<!-- If a browser cannot find an image , it will display the value of the alt attribute-->
+
+
+<img src="img_girl.jpg" alt="girl in a jacket" style="width:50px;height:600px;" />
```
Comment (msindia-learning) [inline PrabhaK/01 - HTML/01 - Basics/03 - Links & Images/Images.html:9]: img_girl.jpg - Image file missing

File: PrabhaK/01 - HTML/01 - Basics/07 - Id & Classes/ID-Classes.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    #header{
+        background-color:darkorchid;
+    }
+</style>
+
+<h1> My <span class="note">Important </span> Heading</h1>
+<p>This is some<span class="note">Important</span>Text.</p>
+
+<script>
+    function myfunction()
```
Comment (msindia-learning) [inline PrabhaK/01 - HTML/01 - Basics/07 - Id & Classes/ID-Classes.html:36]: purpose for this function?

File: PrabhaK/01 - HTML/01 - Basics/10 - Head & Layouts/Layouts.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            clear:both;
+
+        }
+        footer{
+            background-color:#777;
+            padding:10px;
+            text-align:center;
+            color:white;
+        }
+        @media(max-width:600px);{
```
Comment (msindia-learning) [inline PrabhaK/01 - HTML/01 - Basics/10 - Head & Layouts/Layouts.html:86]: Purpose for this?

**Review by msindia-learning — COMMENTED** [review]
Please have a look.

File: PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Paragraphs.html
Comment (prabha0710) [reply PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Paragraphs.html:1]: Html Template Added.

File: PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html
Comment (prabha0710) [reply PrabhaK/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html:1]: h2, h6 added.

File: PrabhaK/01 - HTML/01 - Basics/02 - Quotations & Comments/Quotations.html
Comment (prabha0710) [reply PrabhaK/01 - HTML/01 - Basics/02 - Quotations & Comments/Quotations.html:1]: Done Sir

File: PrabhaK/01 - HTML/01 - Basics/07 - Id & Classes/ID-Classes.html (hunk 2 above)
Comment (prabha0710) [reply PrabhaK/01 - HTML/01 - Basics/07 - Id & Classes/ID-Classes.html:36]: Use of The class Attribute in JavaScript

File: PrabhaK/01 - HTML/01 - Basics/10 - Head & Layouts/Layouts.html (hunk 3 above)
Comment (prabha0710) [reply PrabhaK/01 - HTML/01 - Basics/10 - Head & Layouts/Layouts.html:86]: My Mistake Sir.

**Review by msindia-learning — APPROVED** [review]

**Pull request merged**